### DISCUSSION TITLE
Provide dedicated buttons for circle clicking on osu!lazer for Android

### DISCUSSION THREAD

**Discussion by CoolGuy12490** [ppy/osu#18857] [Ideas] [answered]
We need to perform streams to do 4*+ maps.

**Comment by frenzibyte** [comment, marked as answer]
It would be quite helpful if you describe in details what you're suggesting here. But guessing from the title, I suppose you're looking for #4201/#4195.

**Reply by CoolGuy12490** [reply]
You (collaborating with the developers) can add two buttons that do what the mouse buttons/z and x do.
In settings under the _Input_ tab, there can be a setting where the buttons will go with a switch between **Left** and  **Right**.

This input method should be in osu!standard.

**Reply by frenzibyte** [reply]
You mean something like an on-screen controller? if so, that's what #4195 aims to add.
